crates/forge/README.md

```diff
@@ -72,33 +72,33 @@ function testDoubleWithFuzzing(uint256 x) public {
 
 ## Features
 
--   [ ] test
-    -   [x] Simple unit tests
-        -   [x] Gas costs
-        -   [x] DappTools style test output
-        -   [x] JSON test output
-        -   [x] Matching on regex
-        -   [x] DSTest-style assertions support
-    -   [x] Fuzzing
-    -   [ ] Symbolic execution
-    -   [ ] Coverage
-    -   [x] HEVM-style Solidity cheatcodes
-    -   [ ] Structured tracing with abi decoding
-    -   [ ] Per-line gas profiling
-    -   [x] Forking mode
-    -   [x] Automatic solc selection
--   [x] build
-    -   [x] Can read DappTools-style .sol.json artifacts
-    -   [x] Manual remappings
-    -   [x] Automatic remappings
-    -   [x] Multiple compiler versions
-    -   [x] Incremental compilation
-    -   [ ] Can read Hardhat-style artifacts
-    -   [ ] Can read Truffle-style artifacts
--   [x] install
--   [x] update
--   [ ] debug
--   [x] CLI Tracing with `RUST_LOG=forge=trace`
+- [ ] test
+  - [x] Simple unit tests
+    - [x] Gas costs
+    - [x] DappTools style test output
+    - [x] JSON test output
+    - [x] Matching on regex
+    - [x] DSTest-style assertions support
+  - [x] Fuzzing
+  - [ ] Symbolic execution
+  - [ ] Coverage
+  - [x] HEVM-style Solidity cheatcodes
+  - [ ] Structured tracing with abi decoding
+  - [ ] Per-line gas profiling
+  - [x] Forking mode
+  - [x] Automatic solc selection
+- [x] build
+  - [x] Can read DappTools-style .sol.json artifacts
+  - [x] Manual remappings
+  - [x] Automatic remappings
+  - [x] Multiple compiler versions
+  - [x] Incremental compilation
+  - [ ] Can read Hardhat-style artifacts
+  - [ ] Can read Truffle-style artifacts
+- [x] install
+- [x] update
+- [ ] debug
```

Comment:
same decent support for debug

Not sure if https://github.com/foundry-rs/foundry/blob/7cf0e15da05cccf10331fc5ecb6227dd6795b2ee/crates/forge/README.md#consolelog is still relevant in scope of v1.0, we plan to remove it

For `Daptools feature parity` https://github.com/foundry-rs/foundry/tree/zerosnacks/chore-update-readme/crates/forge#dapptools-feature-parity I think we only missing point 6 Symbolic EVM

Reply:
Removed the outdated docs for now, users should just use the Foundry book